### PR TITLE
add link to macos/ios configuration profile

### PR DESCRIPTION
**Pull request by georgehrke**
Follow up for https://github.com/nextcloud/server/issues/1434#issuecomment-459294539

### CODE REVIEW

File: lib/Controller/WizardController.php [hunk 1]
```diff
@@ -74,7 +74,9 @@ public function show() {
 			'desktop'      => $this->config->getSystemValue('customclient_desktop', $this->theming->getSyncClientUrl()),
 			'android'      => $this->config->getSystemValue('customclient_android', $this->theming->getAndroidClientUrl()),
 			'ios'          => $this->config->getSystemValue('customclient_ios', $this->theming->getiOSClientUrl()),
-			'appStore' => $this->config->getSystemValue('appstoreenabled', true),
+			'appStore'     => $this->config->getSystemValue('appstoreenabled', true),
+			'useSSL'       => $this->request->getHttpProtocol() === 'https',
```
Comment (MorrisJobke) [inline lib/Controller/WizardController.php]: It's TLS hopefully.

Comment (MorrisJobke) [inline lib/Controller/WizardController.php]:
```suggestion
			'useTLS'       => $this->request->getHttpProtocol() === 'https',
```

File: templates/page.clients.php [hunk 2]
```diff
@@ -65,6 +65,13 @@
 				<span class="icon icon-files-dark"></span>
 				<?php p($l->t('Access files via WebDAV')); ?>
 			</a>
+			<?php if($_['useSSL']): ?>
```
Comment (MorrisJobke) [inline templates/page.clients.php]:
```suggestion
			<?php if($_['useTLS']): ?>
```

File: templates/personal-settings.php [hunk 3]
```diff
@@ -48,6 +48,8 @@
 	</div>
 	<?php
 		$appPasswordUrl = \OC::$server->getURLGenerator()->linkToRoute('settings.PersonalSettings.index', ['section' => 'security']);
+		$macOSProfile = \OCP\Util::linkToRemote('dav') . 'provisioning/apple-provisioning.mobileconfig';
+		$usesSSL = \OC::$server->getRequest()->getHttpProtocol() === 'https';
```
Comment (MorrisJobke) [inline templates/personal-settings.php]:
```suggestion
		$usesTLS = \OC::$server->getRequest()->getHttpProtocol() === 'https';
```

File: templates/personal-settings.php [hunk 4]
```diff
@@ -72,5 +74,12 @@
 				 src="<?php p(image_path('files', 'folder.svg')); ?>" />
 			<?php p($l->t('Access files via WebDAV'));?>
 		</a>
+		<?php if($usesSSL): ?>
```
Comment (MorrisJobke) [inline templates/personal-settings.php]:
```suggestion
		<?php if($usesTLS): ?>
```

**Comment by juliushaertl** [comment]
@georgehrke Signed-off message is missing on the second commit :wink:

**Pull request merged**